influxql/ast.go

```diff
@@ -5300,9 +5309,16 @@ type Valuer interface {
 	Value(key string) (interface{}, bool)
 }
 
+// ZoneValuer is the interface that specifies the current time zone.
+type ZoneValuer interface {
```

Comment:
Do we really need this interface? Seems like the only place this gets used is

```
+	loc := time.UTC
 +	if v, ok := valuer.(ZoneValuer); ok {
 +		loc = v.Zone()
 +	}
```
Is there any anticipation that there will be anything else that implements `ZoneValuer`?

I guess I've never quite understood why people use create an interface (it seems pretty common) for something where a concrete type would suffice.